src/composables/useTresContextProvider/index.ts

```diff
@@ -182,6 +183,7 @@ export function useTresContextProvider({
     setCameraActive,
     deregisterCamera,
     loop: createRenderLoop(),
+    isTresContext: true,
```

Comment:
Hi @andretchen0 what's the purpose of the `isTresContext`? Couldn't find any reference of usage on your changes

Reply:
Hey @alvarosabu , good catch.

I had started to tackle general-purpose disposal for primitives.

Tres objects have a link to `TresContext` in their `__tres`. If we recurse through it to dispose, we'll end up disposing `Scene` and `Renderer`. So I added `is.tresContext` so we have an easy predicate to check against.

In the end, I decided against putting general purpose disposal in this PR, since it's a standalone function. But I left the `is.tresContext` and the field here.

Would you like me to remove it?

Reply:
Yep let's remove it 👍

Reply:
Hey @alvarosabu 

Ok, I'll go ahead and remove it here.

I'm working on disposal right now and will need to *not* recurse through a `TresContext`. Is it ok with you if I put the field back in that PR?

We *can* duck type a `TresContext` from its existing fields, but I'd prefer to have `isTresContext` since ...

* an `is[...]: true` field is how most THREE objects identify themselves
* duck typing on the existing fields of `TresContext` leaves us open to breaking `is.tresContext` with a refactor that changes its fields. We're unlikely to change `isTresContext: true`, because this is the *only* purpose it serves.

Reply:
Deal